test/src/unit-comparison.cpp

```diff
@@ -205,6 +209,13 @@ TEST_CASE("lexicographical comparison operators")
             {
                 for (size_t j = 0; j < j_values.size(); ++j)
                 {
+                    // Skip comparing indicies 12 and 13, and 13 and 12 in C++20 pending fix
```

Comment:
Can you please add a link to the pending fix?

Reply:
I've added a reference to issue #3207.

This PR actually fixes #3138. The conversation moved on to what is tracked in #3207 which I'll address in part 2.